tests/acceptance/bootstrap/AuthAppContext.php

```diff
@@ -33,6 +33,7 @@
  */
 class AuthAppContext implements Context {
 	private FeatureContext $featureContext;
+	private ResponseInterface $createdAuthAppToken;
```

Comment:
if you are using it to store single token
```suggestion
	private ResponseInterface $lastCreatedToken;
```